docs/cloud/how-to-manage-certificates-in-temporal-cloud.md

```diff
@@ -64,69 +73,28 @@ You can use this tool in two ways:
   This procedure is the easiest because it's independent of your operating system.
 - Follow the README instructions in the [client-only](https://github.com/temporalio/samples-server/tree/main/tls/client-only) directory in our `temporalio/samples-server` repository in GitHub.
 
+:::info
+
+The maximum number of CA certificates in a certificate bundle is 25. The payload size of a certificate bundle (before base64-encoding) is 32 KB.
+
+:::
+
 ## Control authorization for Cloud Namespaces
 
-Because Temporal Cloud uses mTLS for authorization, we recommend that you ensure that a client certificate grants access only to a specific Namespace.
+Because Temporal Cloud uses mTLS for authorization, we recommend that an end-entity certificate be limited to a specific Namespace.
 Temporal Cloud requires full CA chains, so you can achieve authorization in two ways.
 
-### Option 1: Generate a separate certificate for each Namespace
+### Option 1: Issue a separate certificate for each Namespace
 
-Each certificate must belong to a chain up to the CA certificate.
-Temporal uses the CA certificate as the trusted authority for access to your Namespaces.
+Each certificate must belong to a chain up to the root CA certificate.
+Temporal uses the root CA certificate as the trusted authority for access to your Namespaces.
 
 1. Ensure that your certificates meet the [Requirements for client certificates](#requirements-for-client-certificates).
 1. [Add client CA certificates to a Cloud Namespace](/cloud/tcld/namespace/accepted-client-ca/add).
 
 ### Option 2: Create certificate filters for each Namespace
```

Comment:
Namespaces share a common root, but each Namespace has a certificate filter configured

Reply:
(but somehow worded in less words...)